data/json/requirements/toolsets.json

```diff
@@ -15,6 +15,12 @@
     "tools": [ [ [ "forge", 20 ], [ "char_forge", 20 ], [ "oxy_torch", 20 ] ] ]
   },
   {
+    "id": "sewing_standard",
+    "type": "requirement",
+    "//": "Crafting or repair of fabric items",
+    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_wood", 1 ] ] ]
```

Comment:
> Why not the bone needle?

Omission. Example of why I want to use tool sets

> Also why specify wooden needle and tailor kit? Doesn't the system check for subs anyway?

Existing bugs with this approach #17517

> Or make a "SEWING" quality that can consume charge.

Same issue as `WELDING`. Long term goal. Will necessitate further rewrite of crafitng.